### DISCUSSION TITLE
Position of tile card features

### DISCUSSION THREAD

**Discussion by peterhollis** [home-assistant/frontend#21890] [Dashboard] [answered]
The new dashboard design direction is fantastic! 

One small improvement I'd love to see is the ability to control the position of features added to the Tile card. 

Similar to the layout options available for mushroom cards. 

<img width="1121" alt="image" src="https://github.com/user-attachments/assets/7d6c7a50-4fa3-469e-b707-cdba0b000be7">

This would be particularly useful given 2024.9 introduced the ability to control the width of sections.

**Comment by Misiu** [comment]
That would be an awesome feature. It would make the tiles even more compact.
I think, that on small screens, the layout should automatically change (maybe a param for that), so when there is not enough space for the feature it should move to the bottom (on wide screen you would get the look from the right side, but on narrow you would have the look from the left side).
The responsive is something I don't have a strong opinion, but the compact tile card would be great feature.

**Reply by HyperCriSiS** [reply]
I am also really missing this. 
There is currently no card that is really optimal for mobile to create a compact HA Dashboard for lights. Every card misses something and much space is wasted for nothing 🙁😔

**Reply by HyperCriSiS** [reply]
Also no live update with the brightness slider.
How are you supposed to properly control a light if you have to guess how the result will be, lol... 😅

**Comment by jpbede** [comment, marked as answer]
Stay tuned for 2025.3 and listen carefully to the live stream 😉

**Reply by HyperCriSiS** [reply]
That update does not really help much because of several major flaws. :disappointed:
It is not possible to define different positions for different features.
Also if you want to show brightness and color slider inline, they should show up both and share the available space.
Also when using inline and the tile is bigger the main part moves into the middle, it should stay at top and further features below it.
Still no live update, which I don't understand, that's such a basic feature.

**Reply by BBE-FR** [reply]
For a better in line view, the only alternative is mushroom if you want several in line features. You have the other sliders reduced as button and can tap them to adjust color, color-temp or brightness. 

For the live update, using this kind of interfaces for years, I had to play with it to try to understand your point... If you are bothered by the fact that the "update" is send only when you release the slider and not live, I have to admit that I never had any difficulties with that, and even after you point it, I still do not have...

**Reply by HyperCriSiS** [reply]
> For a better in line view, the only alternative is mushroom if you want several in line features. You have the other sliders reduced as button and can tap them to adjust color, color-temp or brightness.
> 
> For the live update, using this kind of interfaces for years, I had to play with it to try to understand your point... If you are bothered by the fact that the "update" is send only when you release the slider and not live, I have to admit that I never had any difficulties with that, and even after you point it, I still do not have...

It is just unnatural. I mean if you use a hardware dimmer, you get an instant result, that's just how it should work imo.

And I really don't understand how someone could make such nice changes with the tile card update and in the same time implementing it SO damn bad, that it makes just 20% use of the possibilities.
